core/templates/components/summary-tile/exploration-summary-tile.component.ts

```diff
@@ -157,6 +159,10 @@ export class ExplorationSummaryTileComponent implements OnInit, OnDestroy {
     }
   }
 
+  isLanguageRTL(): boolean {
+    return this.i18nLanguageCodeService.isCurrentLanguageRTL();
+  }
+
```

Comment:
Why is this added?

Reply:
It was added when the separate .rtl.css file was there. Removed.